File: Modul 6/Soal 3/PRAK603-2410817220018-ANGGRAENI DWI ZAHRA.py
def main():
    n1 = int(input("Masukkan n1: "))
    n2 = int(input("Masukkan n2: "))

    if n1 != n2:
        print("Jumlah tidak sama")
        return

    print(f"Masukkan {n1} angka untuk baris pertama:")
    baris1 = [int(input(f"Angka ke-{i+1}: ")) for i in range(n1)]

    print(f"Masukkan {n2} angka untuk baris kedua:")
    baris2 = [int(input(f"Angka ke-{i+1}: ")) for i in range(n2)]

    hasil = [baris1[i] * baris2[i] for i in range(n1)]
    print("Hasil perkalian: ", ' '.join(map(str, hasil)))

if __name__ == "__main__":
    main()

File: Modul 6/Soal 3/PRAK603-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>

int main () {
    int n1, n2;

    printf("Masukkan n1: ");
    scanf("%d", &n1);
    printf("Masukkan n2: ");
    scanf("%d", &n2);

    if (n1 != n2) {
        printf("Jumlah Tidak Sama \n");
        return 0;
    }

    int baris1[n1];
    int baris2[n2];
    int hasil[n1];

    printf ("Masukkan %d angka untuk baris pertama: \n", n1);
    for (int i = 0; i < n1; i++) {
        printf("Angka ke-%d: ", i + 1);
        scanf("%d", &baris1[i]);
    }
    printf ("Masukkan %d angka untuk baris kedua: \n", n1);
    for (int i = 0; i < n2; i++) {
        printf("Angka ke-%d: ", i + 1);
        scanf("%d", &baris2[i]);
    }

    for (int i = 0; i < n1; i++) {
        hasil[i] = baris1[i] * baris2[i];
    }

    printf("Hasil Perkalian: ");
    for(int i= 0; i < n1; i++) {
        printf("%d", hasil[i]);
        printf(" ");
    }
    return 0;
}